service/templates/admin.html

```diff
@@ -1440,6 +1440,19 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>RPC Port</label>
 										<input type="text" ng-model="Settings.cedar.rpc_port">
 									</md-input-container>
+									<md-input-container class="control" style="width:45%;">
+										<label>SPS URL</label>
+										<input type="text" ng-model="Settings.cedar.sps_url">
+									</md-input-container>
+									<label>Send Performance Data to Cedar</label>
+									<md-radio-group data-ng-model="Settings.cedar.send_to_cedar" layout="row">
+										<md-radio-button data-ng-value="false"></md-radio-button>
+										<md-radio-button data-ng-value="true"></md-radio-button>
+									</md-radio-group>
+									<md-input-container class="control" style="width:45%;">
+										<label>Send Ratio to Cedar (1/x), 0 is off</label>
+										<input type="number" ng-model="Settings.cedar.send_ratio_sps">
+									</md-input-container>
```

Comment:
What are the default values here? Ideally, `send_to_cedar` would default to `true` and `send_ratio_sps` to 0. That way we continue sending data to Cedar and can turn on the new functionality later on.

Reply:
Booleans default to false and ints to 0 -- we can set send_to_cedar to true in the DB, or you can flip this logic to be like, `send_to_cedar_disabled` for which false would be the correct value (we do this for feature flags).